.github/workflows/deploy-to-azure-app-service.yml

```diff
@@ -145,7 +145,7 @@ jobs:
       id: set-up-runtime-option
       if: ${{ inputs.runtime != '' }}
       run: |
-        Write-Output "::set-output name=runtime-option::--runtime=${{ inputs.runtime }}"
+        Set-Output "runtime-option=--runtime" "${{ inputs.runtime }}"
```

Comment:
The equality sign shouldn't be part of the `Set-Output` call, and the parameters don't match the original one.